docs/singleKeyvault/README.md

```diff
@@ -251,7 +251,7 @@ variables:
   flux_recreate: 'false'
   gitops_ssh_url: 'git@ssh.dev.azure.com:v3/myorg/app-cluster-manifests'
   gitops_path: 'prod'
-  gitops_ssh_key: '~/cluster-deployment/keys'
+  gitops_ssh_key_path: '~/cluster-deployment/keys'
```

Comment:
Ditto

Reply:
done

Reply:
Not seeing the changes here. You may have updated another file

Reply:
oh you're right, we had another file that needed updating as well.